src/shared/hooks/helpers.ts

```diff
@@ -233,3 +236,14 @@ export function useMobileScreen() {
 
   return width < MOBILE_BREAKPOINT
 }
+
+export function useTrackEvents() {
+  const location = useLocation()
+  const posthog = usePostHog()
+  const isMobile = useMobileScreen()
+
+  useEffect(() => {
+    posthog?.capture("$pageview", { url: location.pathname, isMobile })
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Do we need this rule here?

Reply:
we can add `useEffectOnce` hook (I think we have it in the extension repo)

Reply:
On second we should add the dependencies to array